app/views/dashboard/show.html.erb

```diff
@@ -2,8 +2,19 @@
   <div class="govuk-grid-column-full">
     <h1 class="govuk-heading-l">User dashboard</h1>
     <p class="govuk-body"> You are logged in</p>
-    <% if @current_user.lead_provider_profile %>
+
+    <% if @current_user.lead_provider_admin? %>
       <%= govuk_link_to "Go to school search", school_search_path %>
     <% end %>
+
+    <% if @current_user.early_career_teacher? %>
+      <h2 class="govuk-heading-l">Your Core Induction Programme</h2>
+      <% if @current_user.early_career_teacher_profile&.core_induction_programme&.course_years&.each do |course_year| %>
```

Comment:
Can we not have a helper / method for this @current_user.early_career_teacher_profile&.core_induction_programme&.course_years&.each . https://en.wikipedia.org/wiki/Law_of_Demeter